<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task management system</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>400</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="vboxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="animated">
       <bool>false</bool>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Программа</string>
    </property>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="actionsTasks">
    <property name="title">
     <string>Задачи</string>
    </property>
    <addaction name="insertTaskAction"/>
    <addaction name="separator"/>
    <addaction name="removeTaskAction"/>
    <addaction name="separator"/>
    <addaction name="insertSubtaskAction"/>
    <addaction name="separator"/>
    <addaction name="taskCompletedAction"/>
    <addaction name="separator"/>
    <addaction name="taskIsNotCompletedAction"/>
   </widget>
   <widget class="QMenu" name="actionsEmployees">
    <property name="title">
     <string>Сотрудники</string>
    </property>
    <addaction name="actionAddEmployee"/>
    <addaction name="separator"/>
    <addaction name="actionRemoveEmployee"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="actionsTasks"/>
   <addaction name="actionsEmployees"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="exitAction">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="insertTaskAction">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Добавить задачу</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I, R</string>
   </property>
  </action>
  <action name="removeTaskAction">
   <property name="text">
    <string>Удалить задачу</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R, R</string>
   </property>
  </action>
  <action name="insertSubtaskAction">
   <property name="text">
    <string>Добавить подзадачу</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="taskCompletedAction">
   <property name="text">
    <string>Задача выполнена</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="taskIsNotCompletedAction">
   <property name="text">
    <string>Задача не выполнена</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionAddEmployee">
   <property name="text">
    <string>Добавить сотрудника</string>
   </property>
  </action>
  <action name="actionRemoveEmployee">
   <property name="text">
    <string>Удалить сотрудника</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
